doc/classes/Control.xml

```diff
@@ -1140,10 +1138,10 @@
 			Sent when the node changes size. Use [member size] to get the new size.
 		</constant>
 		<constant name="NOTIFICATION_MOUSE_ENTER" value="41">
-			Sent when the mouse pointer enters the node.
+			Sent when the mouse cursor enters the control's visible area, that is not concealed behind other Controls or Windows, provided its [member mouse_filter] lets the event reach it and regardless if it's currently focused or not.
```

Comment:
I have to admit I need to use the Eng-Spa dictionary to understand 'concealed.' Could you use a moer common word?

Reply:
"Occluded" is the de-facto technical term for this.

Reply:
Is this statement true though? AFAICT after skimming through the code the sorting based on which these mouse notifcations occur doesn't take z-index into account. Which would mean it's not always a matter of the actual visible areas. :thinking:
https://github.com/godotengine/godot/blob/c3769e46b8a82640e14fbc8fca029787af757536/scene/main/viewport.cpp#L1623-L1693
https://github.com/godotengine/godot/blob/c3769e46b8a82640e14fbc8fca029787af757536/scene/main/viewport.cpp#L1381-L1389
https://github.com/godotengine/godot/blob/da81ca62a5f6d615516929896caa0b6b09ceccfc/scene/gui/control.h#L152-L160
https://github.com/godotengine/godot/blob/da81ca62a5f6d615516929896caa0b6b09ceccfc/scene/main/node.cpp#L1759-L1813

Reply:
Thank you for pointing out the z-index. I did miss that.

Now I have been trying to describe the behavior in a way that includes all details but coming up with a concise way to do that is really hard. I hope, the current version is acceptable. I will gladly accept suggestions how to describe this in a better way.